tests_integration/test_data/projects/project_definition_v2/snowflake.yml

```diff
@@ -21,7 +21,7 @@ entities:
       - sql_script: scripts/post_pkg_deploy.sql
   app:
     type: application
-    name: my_app_<% ctx.env.foo %>
+    identifier: my_app_<% ctx.env.foo %>
```

Comment:
Can you add a test showing that referencing an FQN entity in templating works?

Reply:
Talked with @sfc-gh-melnacouzi and the `fqn` won't work out of the box in templating. We will need `fn.get_fqn(entity)` for templating purposes. Taking this into account I would prefer to add such functionality in another PR.